lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -266,6 +266,7 @@ sub start
         update_user_directory => ( not $self->{workers} ),
         enable_media_repo     => ( not $self->{workers} ),
         run_background_tasks_on  => ( $self->{workers} ? "background_worker1" : "master" ),
+        worker_to_update_user_directory  => ( $self->{workers} ? "user_dir" : "null" ),
```

Comment:
It may be useful to add a comment about keeping `update_user_directory` for backwards compatibility.
Otherwise an enthusiastic developer may try to remove it in the future.